test/integration/targets/docker_swarm_service/tasks/tests/logging.yml

```diff
@@ -0,0 +1,148 @@
+---
+
+- name: Registering service name
+  set_fact:
+    service_name: "{{ name_prefix ~ '-options' }}"
```

Comment:
You could of course also use a different suffix here, such as `-logging` :)

Reply:
Good catch, totally missed that.